metafacture-linkeddata/src/main/java/org/metafacture/linkeddata/RdfMacroPipe.java

```diff
@@ -57,6 +59,43 @@ public void setAutoAddedSubject(final String autoAddedSubject) {
         this.autoAddedSubject = autoAddedSubject;
     }
 
+
+            /**
```

Comment:
```suggestion
    /**
```

Reply:
Build fails "Execution failed for task ':metafacture-linkeddata:checkstyleMain'.